aspnetcore/fundamentals/configuration.md

```diff
@@ -42,23 +42,23 @@ The app reads and displays the following configuration settings:
 
 [!code-json[Main](configuration/sample/src/ConfigJson/appsettings.json)]
 
-Configuration consists of a hierarchical list of name-value pairs in which the nodes are separated by a colon. To retrieve a value, you access the `Configuration` indexer with the corresponding item’s key:
+Configuration consists of a hierarchical list of name-value pairs in which the nodes are separated by a colon. To retrieve a value, access the `Configuration` indexer with the corresponding item’s key:
 
 ```csharp
 Console.WriteLine($"option1 = {Configuration["subsection:suboption1"]}");
 ```
 
-To work with arrays in JSON-formatted configuration sources, use a zero-based array index as part of the colon-separated string used to retrieve the value. For instance, to get the name of the first item in the `wizards` array shown above, use the following code:
+Use an array index with a colon-separated string to access arrays in a JSON-formatted configuration. The following example gets the name of the first item in the preceding `wizards` array:
```

Comment:
with --> in
to access array elements
in ~~a~~ JSON-formatted
~~preceding~~

Doesn't Acrolinx usually say put the goal/purpose first (i.e., the original arrangement of this paragraph)?